docs/HowTo/Migrate-Database.md

```diff
@@ -0,0 +1,50 @@
+---
+description: How to migrate a RocksDB database to a LevelDB2 database.
+---
+
+# Migrate the database
+
+As of v21.5.0, new Teku installations use a LevelDB2 database. Previous Teku versions use
+RocksDB which continue to be supported.
+
+LevelDB2 databases use less memory and are proven to be more stable with Teku. Users can migrate
+Teku nodes running a RocksDB database to a LevelDB2 database if the Teku node runs in [pruning mode].
+
+!!! warning
+
+    Teku nodes running in [archive mode] have to resync from genesis to migrate.
```

Comment:
I'd probably add a note to explain that tools to migrate existing databases without resyncing will be provided in future releases.